notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1614 @@
+{
```

Comment:
<p>Missing links to <code>FoldedDistribution</code>.  The folded distribution class is useful even for non-symmetric base distributions, e.g. <code>FoldedDistribution(StudentT(df, 1.0, scale))</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Oh interesting, thanks for pointing this out @fehiepsi . I guess this means folding is an entirely different kind of "truncation". For example, `FoldedDistribution(Normal(loc=1.0))` is not the same as `TruncatedNormal(loc=1.0, low=0.0)`.
I'll elaborate a bit more in the tutorial. Do you know of any examples/applications where folding a non-symmetric base distribution is useful?

Reply:
i think they are generally used as a way of creating heavy-tailed distributions. for example: https://www.sciencedirect.com/science/article/abs/pii/S0377042720306427#!